run.sh

```diff
@@ -15,5 +15,5 @@ trap 'handle_error' ERR
 
 ./synapse/start.sh
 ./riot/start.sh
-node start.js
+node start.js --travis
```

Comment:
presumably what you really want is `node start.js $@` and then `./run.sh --travis` in https://github.com/matrix-org/matrix-react-sdk/blob/master/.travis-test-riot.sh#L37 ? Although since hopefully this is temporary it's not so important.